root/static/scripts/edit/components/ArtistCreditBubble.js

```diff
@@ -260,11 +260,10 @@ component _ArtistCreditBubble(
     return names;
   }, [names, initialBubbleFocus, initialFocusRef]);
 
-  const allowNameRemoval = React.useMemo(() => {
-    return state.names.reduce((accum, name) => {
-      return accum + (name.removed ? 0 : 1);
-    }, 0) > 1;
-  }, [state.names]);
+  const allowNameRemoval = React.useMemo(
+    () => state.names.filter((n) => !n.removed).length > 1,
```

Comment:
Please double-check this, but this seems like an easier-to-read version of what the `reduce` call was doing.

Reply:
Yeah, that looks better. 👍 I usually default to avoiding array/object allocations where I don't need them, sometimes at the cost of clarity.

Reply:
Heh, I think I similarly default to avoiding `reduce` since I always get confused when I'm reading it. :-)